doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•	**Workload diversity**: Should be capable to handle any VNF , CNF and BareMetal workloads.
+
+•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
+
+•	**Future proof**: Should support known and emerging technology trends e.g. SmartNICs, FPGAs and Programmable Switches, multi-cloud aspects (public/private/hybrid), multi-location aspects (multi-cluster, DCI, etc), Edge realted technologies, etc.
+
+
+<a name="5.1.3"></a> 
+### 5.1.3 Network Layering and Concepts
+
+Cloud and Telco networking are layered, and it is very important to keep the layering dependencies low to enable security, separation and portability in between multiple implementations.
+
+Before we start developing a deep model we need to agree on some foundational concepts and layering that allow decoupling of implementations in between the layers. We will emphasize four concepts in this section
+
+ - Underlay and Overlay Networking concepts
+ - Hardware and Software Infrastructure Layer concepts
+ - Software Defined Networking, SDNu and SDNo concepts
+ - Programmable Networking Fabric concept
+
+<a name="5.1.3.1"></a>
+#### 5.1.3.1 Underlay and Overlay Networking concepts
+
+The ETSI NFV model divides networking into an Underlay and an Overlay Network layer. The purpose with this layering is to ensure separation of the SW Virtualisation tenants (Workload) Overlay Networks from each other, whilst allowing the traffic to flow on the shared Underlay Network in between all Ethernet connected HW units.
+
+The Overlay Networking separation is often done through encapsulation of Tenants' traffic using overlay protocols e.g. through VxLAN or EVPN over the Underlay Network e.g. based on L2 (VLAN) or L3 (IP) networks.
+
+In some instances, the SW Virtualisation Tenants can bypass the Overlay Networking encapsulation to achieve better performance or network visibility/control. A common method to bypass the Overlay Networking encapsulation normally done by the SW Virtualisation Layer, is the VNF/CNF usage of SR-IOV that effectively take over the Physical and Virtual Functions of the NIC directly into the VNF/CNF Tenant. In these cases, the Underlay Networking must handle the separation e.g. through a Virtual Termination End Point (VTEP) that encapsulate the Overlay Network traffic.
+
+> **Note:** Bypassing the Overlay Networking layer is a violation of the basic CNTT decoupling principles, but in some cases unavoidable with existing technologies and standards. Until suitable technologies and standards are developed, CNTT have a set of agreed exemptions that forces the Underlay Networking to handle the bypassed Overlay Networking separation.
```

Comment:
@walterkozlowski @rabi-abdel Do we have a documented Exception for this? In the absence of Fabric Automation, I agree we need to address it, but given I wrote the initial SRIOV exception, I know this topic wasn't addressed. Has the fabric angle been incorporated subsequent to the initial SRIOV exception, or do we need to open an Issue to cover this?

Reply:
@markshostak @rabi-abdel I agree with @markshostak.  I have changed to "Until suitable technologies and standards are developed, CNTT needs to have an exemption that forces the Underlay Networking to handle the bypassed Overlay Networking separation." We will create a PR for this.